src/PerformanceTests/ArtifactsParser/Program.cs

```diff
@@ -23,7 +23,15 @@ static void Main(string[] args)
             File.WriteAllText(dst, csv);
             Console.WriteLine("Parsed '{0}' recursively and written to '{1}'", src, dst);
 
-            Process.Start(dst);
+            try
+            {
+                // try to open the file
+                Process.Start(dst);
+            }
+            catch (Exception e)
+            {
+                // ignore exception
```

Comment:
Can you log the exception so that atleast we see what happened?

Reply:
Let's if def then

Reply:
changed to check the platform now. Not sure what will happen on the test windows VM which doesn't have excel installed though.